### PR TITLE
removes duplicate definitions. 

### PR DESCRIPTION
**Pull request by Evankhell561**
before:
before
after:
image

### CODE REVIEW

File: code/__defines/items_clothing.dm [hunk 1]
```diff
@@ -101,11 +101,9 @@
 #define slot_r_ear_str      "slot_r_ear"
 #define slot_belt_str       "slot_belt"
 #define slot_shoes_str      "slot_shoes"
-#define slot_head_str      	"slot_head"
```
Comment (NanakoAC) [inline code/__defines/items_clothing.dm:104]: i added these, don't remove them

File: code/__defines/misc.dm [hunk 2]
```diff
@@ -191,7 +191,6 @@
 #define CELLS 8								//Amount of cells per row/column in grid
 #define CELLSIZE (world.icon_size/CELLS)	//Size of a cell in pixels
 
-#define WORLD_ICON_SIZE 32
```
Comment (NanakoAC) [inline code/__defines/misc.dm:194]: this is very necessary, is it duplicated elsewhere?

File: code/__defines/subsystems.dm [hunk 3]
```diff
@@ -1,22 +1,7 @@
-//Timing subsystem
-//Don't run if there is an identical unique timer active
-//if the arguments to addtimer are the same as an existing timer, it doesn't create a new timer, and returns the id of the existing timer
-#define TIMER_UNIQUE		0x1
-//For unique timers: Replace the old timer rather then not start this one
-#define TIMER_OVERRIDE		0x2
```
Comment (NanakoAC) [inline code/__defines/subsystems.dm:6]: this seems like the place for these to be

**Review by NanakoAC — CHANGES_REQUESTED** [review]
some of these im not sure the correct duplicate has been removed

File: code/__defines/items_clothing.dm (hunk 1 above)
Comment (Evankhell561) [reply code/__defines/items_clothing.dm:104]: Higher up in this dm is this same defined repeated.
image

File: code/__defines/misc.dm (hunk 2 above)
Comment (Evankhell561) [reply code/__defines/misc.dm:194]: image

File: code/__defines/subsystems.dm (hunk 3 above)
Comment (Evankhell561) [reply code/__defines/subsystems.dm:6]: You tell me which of these two files you prefer to keep these defines.
image

Comment (NanakoAC) [reply code/__defines/subsystems.dm:6]: subsystems.dm

**Review by NanakoAC — APPROVED** [review]

**Pull request merged**